site/index.html

```diff
@@ -501,6 +501,37 @@ <h4 id="convolver">Convolver</h4>
                     <button id="play-drums" class="play action"></button>
                 </div>
 
+            <a name="tremolo"></a><h4>Tremolo</h4>
+
+            <pre><code class="language-javascript">
+var tremolo = new Pizzicato.Effects.Tremolo({
+    speed: 4
+    mix: 0.8
+});
+            </code></pre>
+
+            <div class="slider-group">
+                <div class="slider">
+                    <label for="volume-tremolodrums">Volume</label>
```

Comment:
I would use lisp-case for the ```id```s of html elements, something like ```volume-tremolo-drums``` instead of ```volume-tremolodrums```. 

This would also be consistent with other ```id``` attributes such as ```volume-stance-bass```